acquia/blt
# Fixes #3217 to provide acsf-init-verify as part of git hooks and blt …

…validate.

Fixes #3217  
--------

Changes proposed:
---------
(What are you proposing we change?)

- adds validate / acsf to template blt.yml
- adds a acsf-init-validate command to blt validate
- adds a acsf-init-validate command to git hooks

Steps to verify the solution:
-----------
(How does someone verify that this does what you think does?)

1. update your blt.yml file
2. run either blt validate or tests:acsf:validate

Warning, should not be run on non-acsf projects (which is why it defaults to not running).
